### DISCUSSION TITLE
Twind Intellisense breaks auto-import of VS Code

### DISCUSSION THREAD

**Discussion by Andrewnt219** [tw-in-js/twind#178] [Q&A]
Hi everyone, 

I have a problem when Twin Intellisnese is enabled, auto-import on Enter does not work. I installed a fresh next.js project through `yarn create next-app` but still no luck.

**Comment by sastan** [comment]
Could you please create a reproduction repo. That would help me to find the issue. Thanks!

Edit: No need for reproduction.  👇🏾

**Reply by sastan** [reply]
I can confirm this issue. Looking into it.

**Reply by sastan** [reply]
Tracked now in tw-in-js/vscode-twind-intellisense#2
